package.json

```diff
@@ -21,39 +21,42 @@
     "build": "ember build",
     "start": "ember server",
     "test": "npm run verify:supported",
-    "verify:supported": "ember try:each --config-path=config/scenarios-supported.js",
-    "verify:latest": "ember try:each --config-path=config/scenarios-latest.js"
+    "verify:supported": "ember try:each --config-path=config/ember-try.js"
   },
   "dependencies": {
     "ember-cli-babel": "^6.7.1",
     "ember-cli-htmlbars": "^2.0.1",
     "ember-diff-attrs": "^0.2.1",
-    "ember-lifeline": "^1.3.0",
     "ember-singularity-mixins": "^2.1.0",
     "ember-wormhole": "^0.5.2"
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-cli": "2.11.0",
-    "ember-cli-app-version": "^2.0.0",
-    "ember-cli-dependency-checker": "^1.3.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
-    "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-jshint": "^2.0.1",
-    "ember-cli-qunit": "^3.0.1",
-    "ember-cli-shims": "^1.0.2",
+    "broccoli-asset-rev": "^2.7.0",
+    "ember-cli": "^3.3.0",
+    "ember-cli-app-version": "^3.2.0",
+    "ember-cli-dependency-checker": "^3.0.0",
+    "ember-cli-eslint": "^4.2.3",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-qunit": "^4.3.2",
+    "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
-    "ember-cli-test-loader": "^1.1.1",
-    "ember-cli-uglify": "^1.2.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
-    "ember-export-application-global": "^1.0.5",
-    "ember-load-initializers": "^0.6.0",
-    "ember-resolver": "^2.0.3",
-    "ember-source": "~2.16.0",
-    "loader.js": "^4.0.10"
+    "ember-cli-test-loader": "^2.2.0",
+    "ember-cli-uglify": "^2.1.0",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-export-application-global": "^2.0.0",
+    "ember-load-initializers": "^1.1.0",
+    "ember-resolver": "^5.0.1",
+    "ember-source": "~3.3.1",
+    "ember-source-channel-url": "^1.1.0",
+    "ember-try": "^0.2.23",
+    "ember-uuid": "^1.0.1",
+    "eslint-plugin-ember": "^5.2.0",
+    "eslint-plugin-node": "^7.0.1",
+    "loader.js": "^4.7.0"
   },
   "engines": {
-    "node": ">= 0.12.0"
+    "node": "^4.5 || 6.* || >= 8.*"
```

Comment:
We'll need to make sure this is a major version bump if we're changing the minimum supported version of node.